packages/editor/src/components/inserter-sidebar/index.js

```diff
@@ -19,7 +19,10 @@ import { store as preferencesStore } from '@wordpress/preferences';
 import { unlock } from '../../lock-unlock';
 import { store as editorStore } from '../../store';
 
-export default function InserterSidebar() {
+export default function InserterSidebar( {
+	closeGeneralSidebar,
```

Comment:
I was looking at https://github.com/WordPress/gutenberg/pull/60062 and stumbled upon this.

I can't find a reason we pass around these two values that can be retrieved inside `InserterSidebar`. Am I missing something? If not, I think we should do it inside the component.

Reply:
IIRC, the `edit-post|site` package stores still control the sidebar states. We can remove props if they're unified in the `editor` package.

Reply:
Oh.. you're right, yes!